bitnami/ejbca/README.md

```diff
@@ -252,13 +252,9 @@ This chart allows you to set custom Pod affinity using the `affinity` parameter.
 
 As an alternative, you can use of the preset configurations for pod affinity, pod anti-affinity, and node affinity available at the [bitnami/common](https://github.com/bitnami/charts/tree/master/bitnami/common#affinities) chart. To do so, set the `podAffinityPreset`, `podAntiAffinityPreset`, or `nodeAffinityPreset` parameters.
 
-### Configure a private registry
+### Use a different EJBCA version
 
-The `image.tag` parameter allows specifying which image will be pulled for the chart, as described in the [documentation for using a different image version](https://docs.bitnami.com/kubernetes/apps/ejbca/configuration/change-image-version/).
-
-When configuring the `image.tag` parameter to use an image in a private registry, it is necessary to also [specify an image pull secret](https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod). This image pull secret must be passed to the chart at deployment time using the `global.imagePullSecrets` parameter in a `values.yaml` file (it cannot be passed using the `--set` parameter).
-
-Refer to the [chart documentation for more information on configuring a private registry](https://docs.bitnami.com/kubernetes/apps/ejbca/configuration/configure-private-registry/).
+The `image.tag` parameter allows specifying which image will be pulled for the chart. Refer to the [chart documentation for more information on this parameter and how to use it with images from a private registry](https://docs.bitnami.com/kubernetes/apps/ejbca/configuration/change-image-version/).
```

Comment:
Well.. This is not 100% precise. The are actually two parameters to specify what image will be pulled: `image.repository` and `image.tag`.
The `image.tag` specifies the specific version/revision of the image to use.

Reply:
Please note this comment applies to every chart.